test/spec/modules/dailymotionBidAdapter_spec.js

```diff
@@ -288,17 +291,9 @@ describe('dailymotionBidAdapterTests', () => {
     expect(reqData.appStoreUrl).to.eql(bidderRequestData.ortb2.app.storeurl);
     expect(reqData.request.auctionId).to.eql(bidRequestData[0].auctionId);
     expect(reqData.request.bidId).to.eql(bidRequestData[0].bidId);
-    expect(reqData.request.mediaTypes.video.api).to.eql(bidRequestData[0].mediaTypes.video.api);
-    expect(reqData.request.mediaTypes.video.mimes).to.eql(bidRequestData[0].mediaTypes.video.mimes);
-    expect(reqData.request.mediaTypes.video.minduration).to.eql(bidRequestData[0].mediaTypes.video.minduration);
-    expect(reqData.request.mediaTypes.video.maxduration).to.eql(bidRequestData[0].mediaTypes.video.maxduration);
-    expect(reqData.request.mediaTypes.video.protocols).to.eql(bidRequestData[0].mediaTypes.video.protocols);
-    expect(reqData.request.mediaTypes.video.skip).to.eql(bidRequestData[0].mediaTypes.video.skip);
-    expect(reqData.request.mediaTypes.video.skipafter).to.eql(bidRequestData[0].mediaTypes.video.skipafter);
-    expect(reqData.request.mediaTypes.video.skipmin).to.eql(bidRequestData[0].mediaTypes.video.skipmin);
-    expect(reqData.request.mediaTypes.video.startdelay).to.eql(bidRequestData[0].mediaTypes.video.startdelay);
-    expect(reqData.request.mediaTypes.video.w).to.eql(bidRequestData[0].mediaTypes.video.w);
-    expect(reqData.request.mediaTypes.video.h).to.eql(bidRequestData[0].mediaTypes.video.h);
+
+    expect(reqData.request.mediaTypes.video).to.eql(bidRequestData[0].mediaTypes.video);
+
```

Comment:
isn't it better to have the unit test covering each attribute than the full object? then when one attribute is failing we will have the exact attribute where an issue is occuring instead of the whole object?

Reply:
Sounds good, will revert this

Reply:
Like this one:
<img width="859" alt="Capture d’écran 2024-05-22 à 14 37 19" src="https://github.com/dailymotion-oss/Prebid.js/assets/11768029/c5ade6b2-e482-4522-9d1e-d018a14a5396">